CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 - Client factories for Buzz.
 
+### Changed
+
+- **[BC] PluginClientFactory returns an instance of `Http\Client\Common\PluginClient`** (see [php-http/client-common#14](https://github.com/php-http/client-common/pull/14))
+- Plugins are loaded from their new packages, fall back to `php-http/plugins` if necessary
+
```

Comment:
should we say that version 2 of the bundle will not automatically require the plugins that moved to separate repositories? i am not sure if we want to do that actually - cache, logger and stopwatch all depend on things that symfony depends on anyways, so there is no cost in requiring them in the bundle, right? maybe we even want to explicitly require those new plugins rather than the BC repository.